host/xtest/regression_1000.c

```diff
@@ -1934,3 +1934,30 @@ static void xtest_tee_test_1025(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1025, xtest_tee_test_1025,
 		"Test __thread attribute");
+
+static void xtest_tee_test_1026(ADBG_Case_t *c)
+{
+	TEEC_Session session = { 0 };
+	uint32_t ret_orig = 0;
+
+	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
+			xtest_teec_open_session(&session, &os_test_ta_uuid,
+				NULL, &ret_orig)))
+		return;
+
+	Do_ADBG_BeginSubCase(c, "Before dlopen()");
+	(void)ADBG_EXPECT_TEEC_SUCCESS(c,
+		TEEC_InvokeCommand(&session, TA_OS_TEST_CMD_DL_PHDR, NULL,
+				   &ret_orig));
+	Do_ADBG_EndSubCase(c, "Before dlopen()");
+
+	Do_ADBG_BeginSubCase(c, "After dlopen()");
+	(void)ADBG_EXPECT_TEEC_SUCCESS(c,
+		TEEC_InvokeCommand(&session, TA_OS_TEST_CMD_DL_PHDR_DL, NULL,
+				   &ret_orig));
+	Do_ADBG_EndSubCase(c, "After dlopen()");
+
+	TEEC_CloseSession(&session);
+}
+ADBG_CASE_DEFINE(regression, 1026, xtest_tee_test_1026,
+		"Test dl_iterate_phdr()");
```

Comment:
bad indentation. possibly also in previous commit(s).

Reply:
Will fix